src/Symfony/Component/Ldap/Adapter/QueryInterface.php

```diff
@@ -23,6 +23,10 @@
     const DEREF_FINDING = 0x02;
     const DEREF_ALWAYS = 0x03;
 
+    const SCOPE_BASE = 0x00;
```

Comment:
Do not use integer constants here unless they are actually part of the specification. Here, please use string values.

I.e.

```php
const SCOPE_BASE = 'base';
const SCOPE_ONELEVEL = 'one_level';
const SCOPE_SUBTREE = 'subtree';
```